nautobot/extras/models/jobs.py

```diff
@@ -334,6 +335,101 @@ def get_absolute_url(self):
         return reverse("extras:job_detail", kwargs={"slug": self.slug})
 
 
+@extras_features("graphql")
+class JobHook(BaseModel, ChangeLoggedModel):
```

Comment:
Any reason this shouldn't be a full-featured `OrganizationalModel`? I assume this is just patterned after WebHook, but we might as well have this support custom fields and relationships out of the box.

Reply:
Agreed. I meant to ask about that